views/macros/signaturePad.njk

```diff
@@ -0,0 +1,103 @@
+{#
```

Comment:
Macro-ized the signature pad, so that we can use it again for the medical professional consent.

cc @dsamojlenko

Reply:
beauty